Client.Linq/Internal/Expressions/String/EndsWith.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions.String
+{
+    internal class EndsWith : IExpressionPart
```

Comment:
Please rename to `HasSuffix`
```suggestion
    internal class HasSuffix : IExpressionPart
```